server.rb

```diff
@@ -23,21 +18,15 @@ def review_label!(repository, issue)
 
 def comment_created_message(repository, issue, comment)
   if comment.matches?(/p\w\wg/i)
-    if issue.owner?(comment.user)
-      message = "wants input on their issue"
-    else
-      message = "wants input on this issue"
-    end
+    message = "Ping :ping:"
+  elsif comment.matches?(/\+1/i)
+    message = "Thumbs up :+1:"
   elsif comment.matches?(/lgtm/i)
-    if issue.owner?(comment.user)
-      message = "thinks their issue is great"
-    else
-      message = "has a positive opinion about this"
-    end
+    message = "Looks good :check:"
   end
 
   if message
-    return "#{team_name_for_project(repository)}#{issue} #{comment.user} #{message}\n```#{comment.body.slice(0...100)}```"
+    return "> #{repository} #{issue} #{comment.user}: #{message}\n>>>#{comment.body.slice(0...100)}..."
```

Comment:
Display content in quoted blocks instead of backtick blocks to allow hilighted nicks and emojis to come through